reference/src/taimpl/include/porting/memory.h

```diff
@@ -121,6 +121,7 @@ void* memory_memset_unoptimizable(
         uint8_t value,
         size_t size);
 
+#ifndef DISABLE_SVP
```

Comment:
Shouldn't the declaration above be #ifdef'ed:
void memory_secure_free(void* buffer);